### ISSUE TITLE
Python API recursive PUT

### ISSUE THREAD

**Issue by austinmw** [NVIDIA/aistore#118]
Hi, what is the Python API analogous command to the following?

```python
# put the downloaded dataset in the created AIS bucket
! ais object put -r -y <path_to_dataset> ais://caltech256/
```

**Comment by alex-aizman** [comment]
Doesn't exist yet. We are currently working to uplift the entire Python-based API/SDK. 

Separately, there's also something called **promote** that may happen to be useful (but again, with no Python support yet):

* https://aiatscale.org/blog/2022/03/17/promote

**Comment by austinmw** [comment]
Okay, thanks for your response!

**Comment by aaronnw** [comment]
Put and Promote are both included in the Python SDK now (v1.1.1). You can see examples here: https://github.com/NVIDIA/aistore/blob/master/python/examples/sdk/uploading-objects.ipynb and docs specifically for recursive put here: https://github.com/NVIDIA/aistore/blob/master/docs/python_sdk.md#put_files
